app/models/hbx_enrollment.rb

```diff
@@ -61,11 +61,19 @@ def household=(hh)
                             )
   WAIVED_STATUSES     = %w(inactive renewing_waived)
 
+  INDIVIDUAL_KIND     = "individual".freeze
+  COVERALL_KIND       = "coverall".freeze
+  GROUP_KINDS         = %w[employer_sponsored employer_sponsored_cobra].freeze
+  OTHER_KINDS         = %w[unassisted_qhp insurance_assisted_qhp streamlined_medicaid emergency_medicaid hcr_chip].freeze
+
   ENROLLED_AND_RENEWAL_STATUSES = ENROLLED_STATUSES + RENEWAL_STATUSES
 
   ENROLLED_RENEWAL_WAIVED_STATUSES = ENROLLED_STATUSES + RENEWAL_STATUSES + WAIVED_STATUSES
   TERM_REASONS = %w[non_payment voluntary_withdrawl retroactive_canceled].freeze
 
+  IVL_KINDS = [INDIVIDUAL_KIND, COVERALL_KIND].freeze
+  INSURANCE_KINDS = [IVL_KINDS, GROUP_KINDS, OTHER_KINDS].freeze
```

Comment:
@ATBull81 Is this expected? This returns an array of arrays which is not expected.

Reply:
[1fc80ae](https://github.com/ideacrew/enroll/pull/3518/commits/1fc80aef30e61fda106a5f5a34e2b954d891be59)